tests/NLog.UnitTests/Targets/DefaultJsonSerializerTestsBase.cs

```diff
@@ -400,39 +400,71 @@ public void SerializeValueTuple_Test()
         [Fact]
         public void SerializeAnonymousObject_Test()
         {
-            var object1 = new { Name = "test name" };
+            var object1 = new { Id = 123, Name = "test name" };
             var actual = SerializeObject(object1);
-            Assert.Equal("{\"Name\":\"test name\"}", actual);
+            Assert.Equal("{\"Id\":123, \"Name\":\"test name\"}", actual);
         }
 
 #if DYNAMIC_OBJECT
 
+        [Fact]
+        public void SerializeExpandoObject_Test()
```

Comment:
Is this test related to "ObjectReflectionCache - Extracted from JsonSerializer for use in XmlLayout"?

Reply:
> Improves support for objects implementing `IDictionary<string,object>` (Ex Expando-objects)